cypress-shared/support/location-availability/support.js

```diff
@@ -0,0 +1,88 @@
+import { FAIL_ON_STATUS_CODE, VTEX_AUTH_HEADER } from '../common/constants'
+import selectors from '../common/selectors'
+import { updateRetry } from '../common/support'
+import { getPickupPoints, deletePickupPoint } from './pickup-points.api'
+
+export function verifyUpdatedAddress(postalCode) {
+  it('Verify on click to postal code it opens the location popup', () => {
+    cy.get(selectors.AvailabilityHeader).click()
+    cy.get(selectors.AddressModelLayout).should('be.visible')
+    cy.get(selectors.countryDropdown).select('USA')
+    cy.get(selectors.addressInputContainer).eq(0).clear().type(postalCode)
+    cy.waitForGraphql('address', selectors.SaveButton)
+    cy.once('uncaught:exception', () => {
+      return false
+    })
+  })
+  it('Verify updated address is shown in the screen', updateRetry(3), () => {
+    cy.getVtexItems().then((vtex) => {
+      cy.intercept('POST', `${vtex.baseUrl}/**`, (req) => {
+        if (req.body.operationName === 'updateOrderFormShipping') {
+          req.continue()
+        }
+      }).as('updateOrderFormShipping')
+      cy.get(selectors.AvailabilityHeader).should('have.text', postalCode)
+      cy.wait('@updateOrderFormShipping', { timeout: 20000 })
+    })
+    cy.get(selectors.AddtoCart).contains('Add to cart').click({ force: true })
+    cy.get(selectors.ProceedtoCheckout).click()
+    cy.get(selectors.orderButton).should('be.visible').click()
+  })
+}
+
+export function addPickUpPoint(pickPointName, pickUpId) {
+  cy.get(selectors.AddPickUpButton).click()
+  cy.get(selectors.PickUpPointName).clear().type(pickPointName)
+  cy.get(selectors.PickUpPointId).should('be.visible').type(pickUpId)
+  cy.get('select')
+    .select('United States of America')
+    .should('have.value', 'USA')
+  /* eslint-disable cypress/no-unnecessary-waiting */
+  cy.wait(1000)
+  /* eslint-disable cypress/no-unnecessary-waiting */
+  cy.get(selectors.PickUpAddress)
+    .type('1481 Maple View Dr,Promona,CA,USA', { delay: 50 })
+    .wait(500)
+    .type('{downarrow}{enter}')
+  cy.get(selectors.CheckBox).click()
+  cy.get(selectors.WorkStartTime).type('10:00')
+  cy.get(selectors.WorkEndTime).type('19:00')
+  cy.get(selectors.SaveChanges).click()
+  cy.get(selectors.ChangesSaved).should('be.visible').contains('Changes saved')
+  cy.get('.vtex-button > .flex > .ml3').click()
```

Comment:
Move this selectors file and add should be visible this will ensure button is visible